### PR TITLE
Feature/create category view

### PR DESCRIPTION
**Pull request by mthanh2209**

### CODE REVIEW

File: javascript-practice/src/javascripts/template/footer.js [hunk 1]
```diff
@@ -1,32 +1,31 @@
...
-	constructor() { }
+	constructor() {}
 
 	/**
 	 * HTML template for rendering the footer.
 	 * @returns {HTMLElement} HTML element for the footer.
 	 */
-	static renderFooter = (
-		`
+	static renderFooter = (data) => {
```
Comment (hungnguyenhuu-agilityio) [inline javascript-practice/src/javascripts/template/footer.js:15]: Could you help to add comments for the data structure you will put into this one?

Comment (mthanh2209) [reply javascript-practice/src/javascripts/template/footer.js:15]: I added comments

Comment (hungnguyenhuu-agilityio) [reply javascript-practice/src/javascripts/template/footer.js:16]: Could you add an example for this one?

Comment (mthanh2209) [reply javascript-practice/src/javascripts/template/footer.js:16]: I fixed it, Pls recheck help me

File: javascript-practice/src/javascripts/views/layout.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+		this.renderLayout();
+	}
+
+	//----- RENDERING -----//
+
+	/**
+	 * Renders the header and footer elements.
+	 * @param {Object} data  - data to render the header and footer.
+	 */
+	renderLayout = async () => {
```
[the remark this answers is not in the document]
Comment (mthanh2209) [reply javascript-practice/src/javascripts/views/layout.js:19]: Thank you, I fixed it @hungnguyenhuu-agilityio

**Pull request merged**